### PR TITLE
fix basic_incident_presented

### PR DESCRIPTION
**Pull request by matthyx**
### **User description**
@Bezbran did it ever succeed?

___

### **PR Type**
enhancement

___

### **Description**
- Improved code readability and consistency in `kubectl_wrapper.py` by cleaning up formatting and enhancing logging outputs.
- Extended timeout values and updated logging in `incidents.py` to enhance the robustness and traceability of incident handling tests.
- Fixed logical errors in application profile verification to ensure accurate status checks.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Enhancement
</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>kubectl_wrapper.py</strong><dd><code>Code Formatting and Logging Enhancements in Kubernetes Wrapper</code></dd></summary>
<hr>

infrastructure/kubectl_wrapper.py
<li>Improved formatting by removing unnecessary whitespace and adjusting <br>line breaks.<br> <li> Fixed inconsistent return statement formatting in several methods.<br> <li> Enhanced logging statements to include dynamic data.<br>

</details>
    

  </td>
  <td><a href="https://github.com/armosec/system-tests/pull/347/files#diff-217696a861da873133ae25152ab3510b116a9fe1b39b312fb0bbdbc566347ea9">+9/-9</a>&nbsp; &nbsp; &nbsp; </td>
</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>incidents.py</strong><dd><code>Enhancements and Bug Fixes in Incident Handling Tests</code>&nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tests_scripts/runtime/incidents.py
<li>Updated logging to include dynamic information for better <br>traceability.<br> <li> Extended timeout settings in incident verification methods to improve <br>reliability.<br> <li> Corrected logical conditions in application profile verification.<br>

</details>
    

  </td>
  <td><a href="https://github.com/armosec/system-tests/pull/347/files#diff-6f3165404f3f21321f01ad4b86d65b7ae44d169e8b0ace497200a7dbe7b4ca90">+16/-16</a>&nbsp; </td>
</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: tests_scripts/runtime/incidents.py [hunk 1]
```diff
@@ -87,19 +87,19 @@ def verify_incident_in_backend_list(self, cluster, namespace, incident_name):
...
         assert k8s_data != None, "Failed to get application profiles"
         assert len(k8s_data) >= len(wlids), f"Failed to get all application profiles {len(k8s_data)}"
-        Logger.logger.info("Application profiles are presented", len(k8s_data))
+        Logger.logger.info(f"Application profiles are presented {len(k8s_data)}")
         ap_wlids = [i.metadata.annotations['kubescape.io/wlid'] for i in k8s_data]
         for i in wlids:
             assert i in ap_wlids, f"Failed to get application profile for {i}"
         # kubescape.io/status: completed, kubescape.io/completion: complete
-        not_complete_application_profiles = [i for i in k8s_data if i.metadata.annotations['kubescape.io/completion'] != 'complete' or i.metadata.annotations['kubescape.io/status'] != 'completed']
+        not_complete_application_profiles = [i for i in k8s_data if i.metadata.annotations['kubescape.io/completion'] != 'complete' and i.metadata.annotations['kubescape.io/status'] != 'completed']
```
Comment (matthyx) [inline tests_scripts/runtime/incidents.py:101]: also this one here should have never passed...

Comment (matthyx) [reply tests_scripts/runtime/incidents.py:101]: https://en.wikipedia.org/wiki/De_Morgan%27s_laws

**Comment by Bezbran** [comment]
the test passed dor me locally few times in a row + it passed against staging in the latest event-ingester pipe.
So yes, it is prettty stable one

File: tests_scripts/runtime/incidents.py [hunk 2]
```diff
@@ -56,25 +56,25 @@ def start(self):
...
         self.wait_for_report(self.verify_application_profiles, wlids=wlids, namespace=namespace)
         time.sleep(6)
         self.exec_pod(wlid=wlids[0], command="ls -l /tmp")
-        
+
         Logger.logger.info("Get incidents list")
-        incs, _ = self.wait_for_report(self.verify_incident_in_backend_list, timeout=30, sleep_interval=5, cluster=cluster, namespace=namespace, incident_name="Unexpected process launched")
-        
-        inc, _ = self.wait_for_report(self.verify_incident_completed,timeout=5*60, sleep_interval=5, incident_id=incs[0]['guid'])        
+        incs, _ = self.wait_for_report(self.verify_incident_in_backend_list, timeout=180, sleep_interval=5, cluster=cluster, namespace=namespace, incident_name="Unexpected process launched")
```
Comment (Bezbran) [inline tests_scripts/runtime/incidents.py:65]: why 3 minutes? it should available in BE within few seconds

File: tests_scripts/runtime/incidents.py (hunk 1 above)
Comment (Bezbran) [reply tests_scripts/runtime/incidents.py:101]: IDK how it passed. but it was

File: tests_scripts/runtime/incidents.py (hunk 2 above)
Comment (matthyx) [reply tests_scripts/runtime/incidents.py:65]: I just tried to wait more and hopefully get it working (naively) but I can revert

**Comment by Bezbran** [comment]
Regarding the APs - I guess your fix is the fix. just I am not sure what can be the issue.
Regarding the BE waiting period, let's keep it on seconds, not minutes.

**Comment by matthyx** [comment]
> Regarding the APs - I guess your fix is the fix. just I am not sure what can be the issue. Regarding the BE waiting period, let's keep it on seconds, not minutes.

done

**Pull request merged**